### ISSUE TITLE
Announcement: New version coming soon

### ISSUE THREAD

**Issue by icleitoncosta** [wppconnect-team/wppconnect#2302]
Hello everyone, I'm very excited to announce that we are working on version 2.0 of wppconnect.

This version will use Playwright as the browser orchestrator, allowing the use of different types of browsers. We've seen some performance improvements and also a significant enhancement in the project's ease of use. I'll share more details about it soon.

What I need to know from you right now is:

What would you like to see in version 2 of wppconnect?

**Comment by Saifallak** [comment]
- Resource consumption of each session?

**Comment by icleitoncosta** [comment]
> * Resource consumption of each session?

Yes, definitely, I'm studying a way to significantly reduce the browser's consumption, so that WhatsApp itself consumes very few resources, only what's necessary, along with a configuration to change that.

**Comment by Saifallak** [comment]
> > * Resource consumption of each session?
> 
> Yes, definitely, I'm studying a way to significantly reduce the browser's consumption, so that WhatsApp itself consumes very few resources, only what's necessary, along with a configuration to change that.

Great that would be awesome, but i meant a way to return each session consumption,

I mean session X consumption 200MB Ram 12% CPU
Session Y consumption 400MB ram 1% cpu

So i can monitor which sessions uses more resources.

**Comment by icleitoncosta** [comment]
Ahh, I see. I'll add this to our to-do list. I'm not sure if it's possible via Puppeteer, but we're working behind the scenes on the WPPConnect Manager, which will allow you to manage sessions, create new ones, etc. We can add something to monitor resources just like Grafana does.

**Comment by icleitoncosta** [comment]
> > > * Resource consumption of each session?
> > 
> > 
> > Yes, definitely, I'm studying a way to significantly reduce the browser's consumption, so that WhatsApp itself consumes very few resources, only what's necessary, along with a configuration to change that.
> 
> Great that would be awesome, but i meant a way to return each session consumption,
> 
> I mean session X consumption 200MB Ram 12% CPU Session Y consumption 400MB ram 1% cpu
> 
> So i can monitor which sessions uses more resources.

Ahh, I see. I'll add this to our to-do list. I'm not sure if it's possible via Puppeteer, but we're working behind the scenes on the WPPConnect Manager, which will allow you to manage sessions, create new ones, etc. We can add something to monitor resources just like Grafana does.

**Comment by Saifallak** [comment]
> > > > * Resource consumption of each session?
> > > 
> > > 
> > > Yes, definitely, I'm studying a way to significantly reduce the browser's consumption, so that WhatsApp itself consumes very few resources, only what's necessary, along with a configuration to change that.
> > 
> > 
> > Great that would be awesome, but i meant a way to return each session consumption,
> > I mean session X consumption 200MB Ram 12% CPU Session Y consumption 400MB ram 1% cpu
> > So i can monitor which sessions uses more resources.
> 
> Ahh, I see. I'll add this to our to-do list. I'm not sure if it's possible via Puppeteer, but we're working behind the scenes on the WPPConnect Manager, which will allow you to manage sessions, create new ones, etc. We can add something to monitor resources just like Grafana does.

Great, also maybe use playwright instead of puppeteer https://playwright.dev/  ?

**Comment by icleitoncosta** [comment]
> > > > > * Resource consumption of each session?
> > > > 
> > > > 
> > > > Yes, definitely, I'm studying a way to significantly reduce the browser's consumption, so that WhatsApp itself consumes very few resources, only what's necessary, along with a configuration to change that.
> > > 
> > > 
> > > Great that would be awesome, but i meant a way to return each session consumption,
> > > I mean session X consumption 200MB Ram 12% CPU Session Y consumption 400MB ram 1% cpu
> > > So i can monitor which sessions uses more resources.
> > 
> > 
> > Ahh, I see. I'll add this to our to-do list. I'm not sure if it's possible via Puppeteer, but we're working behind the scenes on the WPPConnect Manager, which will allow you to manage sessions, create new ones, etc. We can add something to monitor resources just like Grafana does.
> 
> Great, also maybe use playwright instead of puppeteer https://playwright.dev/ ?

Yes, yes, we are switching to Playwright now, as it allows for a variety of browsers. It was just out of habit that I mentioned Puppeteer, haha.

**Comment by albertojrigail** [comment]
Hi, a lot of these features would be extremely helpful for us as we're currently experiencing issues that would be fixed by this new update. What's your timeline? Would be helpful to know. Thanks.

**Comment by gekkedev** [comment]
> Support for Proxies

There is a PR to provide this in 1.X:  #2354
